yts/yts.py

```diff
@@ -58,8 +76,8 @@ def get_torrent(self, movie_title, torrent_url):
 
     # Gets the popular downloads from the homepage
     # Returns a dictionary with movie title and rating
-    def get_popluar_downloads(self) -> dict:
-        res = self.make_request(self.url)
+    def get_popular_downloads(self) -> dict:
+        res = make_request(self.url)
```

Comment:
Change the `res = self.make_request(url)` to `res = make_request(url)`.
Link: https://github.com/philippeitis/yts-cli/blob/a8b79be9b61a77fdd60c4d01c4f4437aae6af4a5/yts/yts.py#L96